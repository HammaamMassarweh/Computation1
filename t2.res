1 INTEGER 2Error 
