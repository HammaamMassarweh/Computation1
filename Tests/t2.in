2
3
obbj

endobj
