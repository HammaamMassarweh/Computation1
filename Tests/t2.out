1 INTEGER 2
2 INTEGER 3
Error o
